src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -238,6 +282,37 @@ export const WorkspaceDashboard = _.flow(
             storageCostEstimate
           ])
         ]),
+        div({ style: styles.header }, [
+          'Tags',
+          h(InfoBox, { style: { margin: '-0.1rem 0 0 0.25rem' } }, [
+            `${getAppName()} is not intended to host personally identifiable information. Do not use any patient identifier including name,
+          social security number, or medical record number.`
+          ])
+        ]),
+        div({ style: { marginBottom: '0.5rem' } }, [
+          h(AsyncCreatableSelect, {
+            value: null,
+            noOptionsMessage: () => 'Enter at least 3 characters to search',
+            allowCreateWhileLoading: true,
+            placeholder: 'Add a tag',
+            onChange: ({ value }) => this.addTag(value),
+            styles: { option: base => ({ ...base, wordWrap: 'break-word' }) },
+            loadOptions: this.getTagSuggestions
+          })
+        ]),
+        div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+          _.map(tag => {
+            return span({ key: tag, style: styles.tag }, [
+              tag,
+              linkButton({
```

Comment:
Nit/follow-up: Removing a tag takes long enough that we should probably disable this to avoid being able to click it twice. Currently, if you double-click it, it will raise an error due to trying to remove a tag that is no longer on the workspace.